### PR TITLE
Fix Screaming Hall Door in Shadow Labyrinth

### PR DESCRIPTION
**Pull request by Xadras**
## 🍰 Pullrequest

Currently the  Screaming Hall Door in Shadow Labyrinth does not open automaticly nor players are able to click on it to open it manually, so it is not possible to get to murmur.
The door now checks every second if a player is in range and if so, it would be opened.

### Proof

https://youtu.be/LujKi3oJAq0?t=559 (no tbc vid, but it is visible when the door should be opened)

### Issues

- None

### How2Test

- None

### Todo / Checklist

- [X] None

### CODE REVIEW

**Comment by killerwife** [comment]
I assume there is no areatrigger in the area?

**Comment by Xadras** [comment]
Exactly, the only areatrigger in there is that one at entrance

File: src/game/AI/ScriptDevAI/scripts/outland/auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp [hunk 1]
```diff
@@ -192,10 +190,56 @@ InstanceData* GetInstanceData_instance_shadow_labyrinth(Map* pMap)
...
+        if (m_doorCheckNearbyPlayersTimer <= diff)
+        {
+            // If player is in 35y range of door, open it if Vorpil boss is done
+            if (m_go->GetInstanceData()->GetData(TYPE_VORPIL) == DONE)
+            {
+                m_go->GetMap()->ExecuteDistWorker(m_go, 35.0f, [&](Player * player)
+                {
+                    m_go->Use(player);
+                    m_doorOpen = true;
+
```
Comment (killerwife) [inline src/game/AI/ScriptDevAI/scripts/outland/auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp]: Remove this empty line,

File: src/game/AI/ScriptDevAI/scripts/outland/auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp [hunk 2]
```diff
@@ -192,10 +190,56 @@ InstanceData* GetInstanceData_instance_shadow_labyrinth(Map* pMap)
     return new instance_shadow_labyrinth(pMap);
 }
 
+
+struct go_screaming_hall_door : public GameObjectAI
+{
+    go_screaming_hall_door(GameObject* go) : GameObjectAI(go)
+    {
+        m_doorCheckNearbyPlayersTimer = 1000;
```
Comment (killerwife) [inline src/game/AI/ScriptDevAI/scripts/outland/auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp]: put the initialization in constructor style:
, m_doorCheckNearbyPlayersTimer(1000) etc

**Comment by killerwife** [comment]
Added some notes, fix them and I will merge the PR.

**Comment by Xadras** [comment]
Thanks for your notes! 
I have to configure my IDE properly soon, so that I can use autoformat again to get rid of those empty lines

**Comment by killerwife** [comment]
A closing note: I changed the constructor to be in one line, we usually have a verticality problem (i.e. too many lines in files)

**Comment by evil-at-wow** [comment]
>  I changed the constructor to be in one line, we usually have a verticality problem (i.e. too many lines in files)

Perhaps that's your opinion. I agree we have lots of files that are too long, but IMHO that's not because of this.

Also, you're going against our code style with this (it's literally in there: ```next line and indent for class field initialization list``` ), which is a very bad signal to contributors. If even the main developers can't follow their own code style, then how do we expect others to follow it? We should be the ones to lead by example, regardless of our own personal preferences.

**Comment by killerwife** [comment]
I kinda agree but I also do "break" a lot of these rules in scripts. I believe I even added one usage of comma operator. Scripts are usually their own beast and it was a pain especially in SpellEffects.cpp (though I added spell scripting now so we should be k there)

Core has plenty of issues like this, and sometimes I might be doing smth that doesnt make sense for the codestyle, but many of my local decisions are due to merge-errors. After 1600 commits ported down and up you see things slightly differently at times... That being said, I love your input :P Do continue.

**Pull request closed without merging**